packages/base-docs/src/Controller/inline-format.controller.ts

```diff
@@ -0,0 +1,165 @@
+import { ITextSelectionRangeWithStyle, ITextSelectionRenderManager } from '@univerjs/base-render';
+import {
+    BooleanNumber,
+    Disposable,
+    getTextIndexByCursor,
+    ICommandInfo,
+    ICommandService,
+    IDocumentBody,
+    ITextRun,
+    IUniverInstanceService,
+    LifecycleStages,
+    OnLifecycle,
+} from '@univerjs/core';
+import { Inject } from '@wendellhu/redi';
+
+import CommonParameter from '../Basics/commonParameter';
+import { SetInlineFormatBoldCommand, SetInlineFormatCommand } from '../commands/commands/inline-format.command';
+import { IRichTextEditingMutationParams, RichTextEditingMutation } from '../commands/mutations/core-editing.mutation';
+import { TextSelectionManagerService } from '../services/text-selection-manager.service';
+
+@OnLifecycle(LifecycleStages.Rendered, InlineFormatController)
+export class InlineFormatController extends Disposable {
```

Comment:
Give it a simple description of its responsibility.

Reply:
+1